examples/ns/src/PLocAssem_VMS_NS_GenAlpha.cpp

```diff
@@ -5,17 +5,17 @@ PLocAssem_VMS_NS_GenAlpha::PLocAssem_VMS_NS_GenAlpha(
         const int &in_nlocbas, const int &in_nqp,
         const int &in_snlocbas,
         const double &in_rho, const double &in_vis_mu,
-        const double &in_beta, const int &elemtype, 
+        const double &in_beta, const FEType &elemtype, 
         const double &in_ct, const double &in_ctauc )
 : rho0( in_rho ), vis_mu( in_vis_mu ),
   alpha_f(tm_gAlpha->get_alpha_f()), alpha_m(tm_gAlpha->get_alpha_m()),
   gamma(tm_gAlpha->get_gamma()), beta(in_beta),
-  CI( (elemtype == 501 || elemtype == 601) ? 36.0 : 60.0 ),
+  CI( (elemtype == FEType::Tet4 || elemtype == FEType::Tet10 || elemtype == FEType::Tet10_v2) ? 36.0 : 60.0 ),
```

Comment:
This line is wrong.  

Here should be:  elemtype == FEType::Tet4 || elemtype == FEType::Hex8